src/Mod/TechDraw/Gui/ViewProviderPage.cpp

```diff
@@ -233,28 +229,27 @@ void ViewProviderPage::setupContextMenu(QMenu* menu, QObject* receiver, const ch
 {
     Gui::ViewProviderDocumentObject::setupContextMenu(menu, receiver, member);
     QAction* act = menu->addAction(QObject::tr("Show drawing"), receiver, member);
-    act->setData(QVariant((int)_SHOWDRAWING));
+    act->setData(QVariant((int)ShowDrawing));
     QAction* act2 = menu->addAction(QObject::tr("Toggle Keep Updated"), receiver, member);
-    act2->setData(QVariant((int)_TOGGLEUPDATE));
+    act2->setData(QVariant((int)ToggleUpdate));

```

Comment:
When changing to `enum class` (now or later on), this needs to be changed to `static_cast<int>(ToggleUpdate)`

Not sure why we use `QVariant`... but that's for another PR.